### ISSUE TITLE
could you give me your blog theme folder.

### ISSUE THREAD

**Issue by errechydyhamza** [flaviocopes/ama#23]
i want to start a personal blog and i hope if you can give me your blog theme folder, 
i know i can get the theme in Hugo website but i don't know how to make things customizable

**Comment by flaviocopes** [comment]
Hi, no 🙂

The theme is highly customized for my own needs and it's not ready for distribution, which would require a lot of work to prepare for.

Also, a website does not have a lot of things to stand out on the internet. The design and organization is one of them, and I spent a lot of time on it - I don't really want other sites to just look like mine.

Plus, you don't need to make it much complicated. Get the simplest design you can find out there, and focus on the writing part.
This is the theme I derived the current design from: https://themes.gohugo.io/ghostwriter/

**Comment by errechydyhamza** [comment]
yeah thank you i find better solution which is blogger template they are amazing

**Comment by TZMMichael** [comment]
>focus on the writing part
It's true~
